sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat/longIntervalStatsbeatMetrics.ts

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  createDefaultHttpClient,
+  createPipelineRequest,
+  HttpMethods,
+} from "@azure/core-rest-pipeline";
+import { diag } from "@opentelemetry/api";
+import {
+  BatchObservableResult,
+  ObservableGauge,
+  ObservableResult,
+} from "@opentelemetry/api-metrics";
+import { Meter } from "@opentelemetry/api-metrics/build/src/types/Meter";
+import { ExportResultCode } from "@opentelemetry/core";
+import {
+  MeterProvider,
+  PeriodicExportingMetricReader,
+  PeriodicExportingMetricReaderOptions,
+} from "@opentelemetry/sdk-metrics";
+import { AzureMonitorExporterOptions, AzureMonitorStatsbeatExporter } from "../../index";
+import * as ai from "../../utils/constants/applicationinsights";
+import {
+  StatsbeatCounter,
+  StatsbeatResourceProvider,
+  STATSBEAT_LANGUAGE,
+  AIMS_URI,
+  AIMS_API_VERSION,
+  AIMS_FORMAT,
+  EU_CONNECTION_STRING,
+  EU_ENDPOINTS,
+  NON_EU_CONNECTION_STRING,
+  CommonStatsbeatProperties,
+  IVirtualMachineInfo,
+  AttachStatsbeatProperties,
+  StatsbeatFeatureType,
+  StatsbeatOptions,
+} from "./types";
+
+const os = require("os");
+
+/**
+ * Presents a singleton pattern by removing public access to the PrivateLongIntervalStatsbeatMetrics constructor.
+ */
+export class LongIntervalStatsbeatMetrics {
+  private static _instance: PrivateLongIntervalStatsbeatMetrics;
+  constructor(options: StatsbeatOptions) {
+    if (!LongIntervalStatsbeatMetrics._instance) {
+      LongIntervalStatsbeatMetrics._instance = new PrivateLongIntervalStatsbeatMetrics(options);
+    }
+  }
+
+  public getInstance() {
+    return LongIntervalStatsbeatMetrics._instance;
+  }
+}
+
+class PrivateLongIntervalStatsbeatMetrics {
```

Comment:
We have some singletons in this project already that are slightly different, it will be good to align with that to have coding approach https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/context/context.ts#L52